x-pack/plugins/endpoint/common/models/event.ts

```diff
@@ -46,3 +47,42 @@ export function parentEntityId(event: ResolverEvent): string | undefined {
   }
   return event.process.parent?.entity_id;
 }
+
+export function eventCategoryDisplayName(event: ResolverEvent): EventCategory {
+  const eventTypeToNameMap = new Map<string, EventCategory | undefined>([
```

Comment:
is `undefined` necessary in this type?

Reply:
It's inert. I removed it. Thanks for the 👁️.